patsy_pineda/oopProduct.py

```diff
@@ -0,0 +1,35 @@
+class Product(object):
+	def __init__(self, price, itemName, weight, brand, cost, reasonStatus, status = "for sale"):
+		self.price = price
+		self.itemName = itemName
+		self.weight = weight
+		self.brand = brand
+		self.cost = cost
+		self.status = status
+		self.displayinfo()
+	def sell(self):
+		self.status = "sold"
+		self.displayinfo()
+		return self
+	def addTax(self, tax):
+		totalprice = (tax*self.price) + self.price
+		print totalprice
+		return self
+	def returnProduct(self, reason):
+		if reasonStatus == "Defective":
+			self.status = "Defective"
+			self.price = 0
+		if reasonStatus == "Like new":
+			self.status = "For sale"
+		if reasonStatus == "Opened":
+			self.status = "Used"
+			self.price = self.price -(self.price * .20)
+		return self
+
```

Comment:
All the code looks awesome. One thing to nitpick, is to be consistent in your naming conventions. If you start naming all your methods using camel case (ex. returnProduct), then keep doing that convention throughout your code. (ex. you have displayinfo at the end, where displayInfo would be more consistent).